app/modules/assets/models.py

```diff
@@ -145,6 +145,46 @@ def update_symlink(self, asset_asset_group_filepath):
 
         return asset_symlink_filepath
 
+    def mime_type_major(self):
+        if not self.mime_type:
+            return None
+        parts = self.mime_type.split('/')
+        return parts[0]
+
+    def is_mime_type_major(self, major):
+        return self.mime_type_major() == major
+
+    # will only set .meta values that can be derived automatically from file
+    # (will not overwrite any manual/other values); silently fails if unknown type for deriving
+    #
+    #  FIXME - this now is a very basic stub -- it is operating on original file and *very* likely fails
```

Comment:
I tghought we'd agreed to standardise on TODO

Reply:
oh yeah?  must have missed that.  they have a bit of semantic difference to me.  but maybe thats just me.  :smile:   i definitely consider `FIXME` to be _must do_ versus `TODO` more a warning of something that probably needs fixing eventually but isnt really a blocker.   obviously very much my own take on that.   i can drop using `FIXME` for sure.   its the one i hope to use less anyway.